Readme.md

```diff
@@ -3,26 +3,21 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T827941)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [Index.razor](./CS/DxBlazorComponentsDefaultSettings/Pages/Index.razor)
-* [MyTextBox.cs](./CS/DxBlazorComponentsDefaultSettings/Components/MyTextBox.cs)
-* [MyDataGrid.cs](./CS/DxBlazorComponentsDefaultSettings/Components/MyDataGrid.cs)
-<!-- default file list end -->
-
-### DevExpress Blazor Components - A way to set predefined settings for the specific component
+# DevExpress Blazor Components - Set predefined settings for the specific component
 
-In the case of a large Web application, you often use controls with the same settings on numerous pages.  It's quite annoying to specify the same settings many times.
-In this example we show how to specify some default settings for all DevExpress Blazor components in your application.
+In case of a large web application, controls usually have the same settings on different pages. The recommended Blazor approach is to specify the component settings in the markup. It is possible to override the **SetParametersAsync** method to specify default settings for Blazor components in your application. However, we do not recommend that you use this approach as it is not natural for Blazor applications. We also do not test this approach to specify predefined settings with all our controls, so you can use it with your own risk.

```

Comment:
```suggestion
Controls usually have the same settings for different pages of a large web application, and we recommend that you specify the component settings in markup. You can also override the **SetParametersAsync** method to specify the default settings for Blazor components in your application. However, we do not recommend that you use this approach because we did not test this approach to specify predefined settings with all our controls.

```

Reply:
- What do you mean when you say `it is not natural`? 
- You do not do something `with your own risk`, you do it **at** your own risk. However, it is unusual to say this in this context since you've already stated that this is an untested approach
- You do not test it, you either **tested** it or **did not** test it since you are making a recommendation here and it refers to a past event

Reply:
1. I mean that it is not a common approach for Blazor. Users who used other dotnet platforms may want to use the way described in this example. But it is not the way how they should do this.
2-3. Actually, the whole point of updating this description is to emphasize that a user can but should not use this approach. Allow me to disagree that the recommendation refers to the past event. We never test specifying predefined settings (before, now, and after). So, it means that some day we can change our component so that this approach breaks and a user's app will not work.
The main idea is that the approach is not good, not that our components lack testing.
Can we rewrite as follows?
```suggestion
Controls usually have the same settings for different pages of a large web application, and we recommend that you specify the component settings in markup. You can also override the **SetParametersAsync** method to specify the default settings for Blazor components in your application. However, we do not recommend that you use this approach because it is not a common approach in Blazor. Use it at your own risk.

```

Reply:
Ok